index.js

```diff
@@ -1,39 +1,40 @@
-const {noopLogger, makePatternForConferenceScan} = require('./lib/utils');
+const { noopLogger, makePatternForConferenceScan, buildSentinels } = require('./lib/utils');
 const Redis = require('ioredis');
 const fs = require('fs');
 
-let JAMBONES_REDIS_CONFIGURATION = process.env.JAMBONES_REDIS_SENTINELS ? {
-  sentinels: process.env.JAMBONES_REDIS_SENTINELS.split(',').map((sentinel) => {
-    let host, port = 26379;
-    if (sentinel.includes(':')) {
-      const arr = sentinel.split(':');
-      host = arr[0];
-      port = parseInt(arr[1], 10);
-    } else {
-      host = sentinel;
-    }
-    return {host, port};
-  }),
-  name: process.env.JAMBONES_REDIS_SENTINEL_MASTER_NAME,
-  ...(process.env.JAMBONES_REDIS_SENTINEL_PASSWORD && {
-    password: process.env.JAMBONES_REDIS_SENTINEL_PASSWORD
-  }),
-  ...(process.env.JAMBONES_REDIS_SENTINEL_USERNAME && {
-    username: process.env.JAMBONES_REDIS_SENTINEL_USERNAME
-  }),
-  ...(process.env.JAMBONES_REDIS_SENTINEL_SERVER_PASSWORD && {
-    sentinelPassword: process.env.JAMBONES_REDIS_SENTINEL_SERVER_PASSWORD
-  }),
-} : {
+let JAMBONES_REDIS_CONFIGURATION = {
   host: process.env.JAMBONES_REDIS_HOST || 'localhost',
   port: process.env.JAMBONES_REDIS_PORT || 6379
 };
+
+let mode = 'single';
+
+if (process.env.JAMBONES_REDIS_SENTINELS) {
+  mode = 'high availability';
```

Comment:
where this mode variable is being used?

Reply:
line 62 - ``Successfully connected to Redis ${mode} mode``